weaviate/users/user_db_deactivator.go

```diff
@@ -0,0 +1,39 @@
+package users
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+
+	"github.com/weaviate/weaviate-go-client/v5/weaviate/connection"
+	"github.com/weaviate/weaviate-go-client/v5/weaviate/except"
+	"github.com/weaviate/weaviate/client/users"
+)
+
+type UserDBDeactivator struct {
+	connection *connection.Connection
+
+	userID string
+}
+
+func (r *UserDBDeactivator) WithUserID(id string) *UserDBDeactivator {
```

Comment:
There is a "revoke_key" option that is missing (I also forgot in the python client)